plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -150,8 +153,9 @@ const generateNextReviewAssignments = async ({
       reviewAssignments[userOrgKey] = {
         reviewerId: userId,
         orgId,
-        stageId: nextStageId,
-        stageNumber: nextStageNumber,
+        stageId,
+        stageNumber,
+        stageDate,
```

Comment:
i think you'll need to add this to `addReviewAssignments` db method

Reply:
Yes, thanks!